README.md

```diff
@@ -1,5 +1,5 @@
 [![PyPI version](https://badge.fury.io/py/semeio.svg)](https://badge.fury.io/py/semeio)
-[![Build Status](https://travis-ci.com/equinor/semeio.svg?branch=master)](https://travis-ci.com/equinor/semeio)
+[![Actions Status](https://github.com/equinor/semeio/workflows/CI/badge.svg?branch=master)](https://github.com/equinor/semeio/actions?workflow=CI)
```

Comment:
According to this https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-a-workflow-name, the following line would be enough for the CI badge
`![Actions Status](https://github.com/equinor/semeio/workflows/CI/badge.svg)`
Should I change it?

Reply:
And do we want it only on branch master or not?